src/Symfony/Component/Translation/Provider/TranslationProviderCollectionFactory.php

```diff
@@ -20,8 +20,9 @@
  */
 class TranslationProviderCollectionFactory
 {
-    private $factories;
-    private $enabledLocales;
+    /** @var ProviderFactoryInterface[] */
```

Comment:
```suggestion
    /** @var iterable<ProviderFactoryInterface> */
```

Reply:
Committed in batch 56b1c3d110f405b279a41d2c64fbbabd3e1629e6